packages/config/config/devices/0x0179/ch-103.json

```
@@ -0,0 +1,186 @@
+{
+	"manufacturer": "ConnectHome",
+	"manufacturerId": "0x0179",
+	"label": "CH-103",
+	"description": "CH-103 Relay",
+	"devices": [
+		{
+			"productType": "0x0041",
+			"productId": "0x0014"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Temperature and Alarm Reports",
+			"maxNodes": 10,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Commands for S1",
+			"maxNodes": 10,
+			"isLifeline": false
+		},
+		"3": {
+			"label": "Commands for S2",
```

How about:
```suggestion
			"label": "Switch S1",
			"maxNodes": 10,
			"isLifeline": false
		},
		"3": {
			"label": "Switch S2",
```
so it is immediately clear these are operated by usinig the switches?